web/src/main/java/com/epam/rft/atsy/web/controllers/NewApplicationController.java

```diff
@@ -43,7 +50,9 @@ public ModelAndView loadPage() {
    */
   @RequestMapping(method = RequestMethod.POST, value = "/secure/new_application")
   public String saveOrUpdate(@Valid @ModelAttribute StateHistoryDTO stateHistoryDTO,
-                             BindingResult result) {
+                             BindingResult result,
+                             HttpServletRequest request,
```

Comment:
For a single http header you can use `@RequestHeader` annotation on a String parameter, see http://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/bind/annotation/RequestHeader.html

and some usage examples here: http://www.logicbig.com/tutorials/spring-framework/spring-web-mvc/spring-mvc-request-header/